src/main/java/Onandon/checkpoint/Checkpoint.java

```diff
@@ -0,0 +1,99 @@
+package Onandon.checkpoint;
+
+import java.io.*;
+import Onandon.module.*;
```

Comment:
Follow Java coding standards which recommend to list down all imports rather than having wildcard imports.